### DISCUSSION TITLE
Welcome to paircode.dev sandbox Discussions!

### DISCUSSION THREAD

**Discussion by Necmttn** [paircodedev/sandbox#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fajarsiddiqfs** [comment]
👋 Hey Happy to join https://paircode.dev discussions here!
Hello everyone, I'm Fajar Siddiq from Singapore. 
Currently, I'm building communities to help makers
You can find me on [Twitter](https://twitter.com/fajarsiddiqFS/status/1398103820488507393)
